pycbc/psd_variation.py

```diff
@@ -0,0 +1,118 @@
+""" PSD Variation """
```

Comment:
Maybe this file should be within the psd package? And simply renamed 'variation' so the namespace would be. 

pycbc.psd.variation

How does that sound?

Reply:
sounds good to me, moved